release_instructions.md

```diff
@@ -0,0 +1,20 @@
+# Checklist for NIXPY release
+
+1. Create branch (if necessary)  
+   For major releases (e.g., 1.2.0, 1.3.0, etc) create a new branch called `vX.Y` (e.g., `v1.2`).
```

Comment:
I am not sure whether  and major and poinr relases are good name choices (yes i am nitpicking)